javascripts/content_scripts/privly.js

```diff
@@ -763,8 +754,6 @@ var privly = {
 
     "use strict";
 
-    return;
```

Comment:
I think this line is here because the function doesn't work on Google Chrome. I didn't remove it because there are copies of privly.js in several other repos and this makes it easier to port the file to other platforms.